src/centralserver/admin-ui/src/views/Members/Member/Details/MemberDetails.vue

```diff
@@ -75,10 +75,34 @@
             >{{ $t('members.member.details.addServer') }}
           </xrd-button>
         </div>
-        <v-card-title class="card-title">Server</v-card-title>
-        <v-divider></v-divider>
-        <v-card-text>SS1</v-card-text>
-        <v-divider class="pb-4"></v-divider>
+
+        <!-- Table -->
+        <v-data-table
+          :loading="loading"
+          :headers="serversHeaders"
+          :items="servers"
+          :search="searchServers"
+          :must-sort="true"
+          :items-per-page="-1"
+          class="elevation-0 data-table"
+          item-key="id"
+          :loader-height="2"
+          hide-default-footer
+        >
+          <template #[`item.namef`]="{ item }">
+            <div class="members-table-cell-name" @click="toDetails('netum')">
```

Comment:
You mentioned that some static data was left as is for now. Is this `netum` here on purpose?

Reply:
No and the whole template is unused. I deleted it now.